widgets/common/src/deploymentsView/table/renderDeploymentRow.tsx

```diff
@@ -26,6 +29,9 @@ const renderDeploymentRow =
                 {Object.entries(deploymentsViewColumnDefinitions).map(([columnId, columnDefinition]) => (
                     <DataTable.Data key={columnId}>{columnDefinition.render(deployment)}</DataTable.Data>
                 ))}
+                {labelsToShow.map(label => {
```

Comment:
I think more accurate would be to say:

```suggestion
                {labelsToShow.map(key => {
```

as in `labelsToShow` we store labels' keys.